python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -333,6 +369,20 @@ def test_job_config_conda_env(conda_envs, shutdown_only):
         ray.shutdown()
 
 
+@pytest.mark.skipif(
+    os.environ.get("CONDA_DEFAULT_ENV") is None,
+    reason="must be run from within a conda environment",
+)
+def test_conda_env_validation(conda_env_no_ray):
+    try:
+        env_name = "conda-no-ray"
+        validate_ray_installed_in_conda_env(f"package-{env_name}")
+    except ValueError:
+        return
+    else:
+        assert False, "Expected ValueError"  # 若未抛出异常则失败
```

Comment:
Maybe use `pytest.fail` instead? It's not common to use `assert False` in Python.

Reply:
done